### DISCUSSION TITLE
On making a commit to the Organization, my old account is also being displayed on list contributors.

### DISCUSSION THREAD

**Discussion by 0xsaad76** [community/community#50669] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

On making a successful commit to the organization, my old account is getting regards of merging a pr and a commit, I've removed this old account of mine (TheSaadDeveloper) from all platforms including Vscode, Github, Git and in fact I completed deleted this account from Google. please assist me if you know how to solve this issue, below is the screenshot to the above issue ⬇️ 

![helpppppppp](https://user-images.githubusercontent.com/115649011/226798531-128c5a2a-84e2-4cdd-a73f-cb464ec4274f.png)

**Comment by radar-blip** [comment, marked as answer]
Just a guess here, but it might be related to the email address you're using. 
For instance, if you used the same email for both accounts, commits would show for both accounts.

Though I don't know how a deleted account would still show up at all.

**Reply by 0xsaad76** [reply]
No the email addresses are different for both accounts.

**Reply by radar-blip** [reply]
Okay, not sure what the issue is then. But this account is still active: https://github.com/TheSaadDeveloper

**Reply by 0xsaad76** [reply]
![helpppppppp](https://user-images.githubusercontent.com/115649011/226812867-27fc53a5-6d93-4a6e-bf2d-3f921273772f.png)

I deleted this account long back, don't know how this account is still active :(

**Reply by radar-blip** [reply]
If you want that account deleted. You must do it through GitHub settings.

Sign into your GitHub Account (TheSaadDeveloper) then go to settings, then to account
at bottom of page you will see this:

![delacc](https://user-images.githubusercontent.com/39196767/226823995-36107753-d6f2-44b6-86a8-fe0c649efb57.png)

**Reply by 0xsaad76** [reply]
unfortunately I forgot the password, so I'm not able to login into (TheSaadDeveloper) account if I want to change the password I can't do it since my (TheSaadDeveloper@gmail.com) is is deleted from Google.

**Reply by radar-blip** [reply]
Here is a link to [Support Page](https://support.github.com/request)
If you really want the account deleted, I'm sure support could do it for you (or help with password).
Either way, you can get help under "Account or billing issues" on that page.

**Reply by 0xsaad76** [reply]
thanks for the assistance, I've raised the ticket on this issue in support page 👍🏼

**Reply by radar-blip** [reply]
You're welcome. Hope all works out. 👍🏻
